### ISSUE TITLE
In group overview there are multiple groups called; "Grid" 

### ISSUE THREAD

**Issue by paulmignot** [withthegrid/.github#3]
# 🐛 Bug report

## Steps to reproduce

Log in to environment and go to groups, there I see an additional list of groups that were not made by the end user

## Expected behaviour

Only see the groups that were creared by the user 

## Actual behaviour

Additional groups 

## Environment

https://app.withthegrid.com/#/e/eyrn9y/groups
https://app.withthegrid.com/#/e/xd2rd4/groups

![image](https://user-images.githubusercontent.com/25848868/100982847-d6a53200-3548-11eb-9515-a5a4752f5182.png)
![image](https://user-images.githubusercontent.com/25848868/100982974-fb010e80-3548-11eb-849b-829c122b50a9.png)

**Comment by paulmignot** [comment]
This also applies to: https://app.withthegrid.com/#/e/6d66md/groups
I think this happens when GIS data is imported?

**Comment by jdbruijn** [comment]
I don't think this is necessary an issue in the application. In general I think the unique identifiers we use are hash ID's, so not the name. This  means that there can be multiple objects, grids in this case, with the same name and seems valid to me. Agreed, not user friendly but that is 

> I think this happens when GIS data is imported?

I think it depends on how the data is imported and what the value of the grid name is.
In the code I see that `Grid` is the default value for grids and I'm not sure whether there is a name field when adding a grid. https://github.com/withthegrid/platform/blob/62517d523a37769641357051c07791518de15f77/src/controllers/graph/add-node.ts#L42
@everhardt Can you elaborate?

FIY I've left it in the support repo as I'm not sure whether it is an application bug or not. If it is we can move it.
